src/templates/app/user/security/vpn_app.jsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+
+const Step = ({
+    circle_number,
+    text,
+    imgSrc,
+}) => (
+    <div className='step'>
+        <div className='circle'>{circle_number}</div>
+        <div className='content gr-row gr-padding-10 gr-row-align-between'>
+            <p>{text}</p>
+            <img className='vpn-app-icon' src={imgSrc} />
```

Comment:
why not put `it.url_for('images/pages/vpn/[...].svg')` here and just pass in the image name?

Reply:
it's up to you, it's more easy to find where images are used if we don't do what I mentioned.